src/dir.props

```diff
@@ -7,4 +7,8 @@
     <SolutionFile Condition="'$(OsEnvironment)'=='Windows_NT'">$([System.IO.Path]::Combine($(SolutionRoot),"MSBuild.sln"))</SolutionFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EnableDocumentationFile)' == 'true'">
+    <DocumentationFile>$(OutputPath)$(TargetName).xml</DocumentationFile>
```

Comment:
Does this need to be in props? It'd be nice to avoid the props-before-the-props-import pattern.